data/althios.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "Ivan Avila",
+  "displayStatsPublic": true,
```

Comment:
We no longer need this `displayStatsPublic` key anymore. So please remove this from your `json`!

```suggestion
```

Reply:
done